test/e2e/lib/pages/accept-invite-page.js

```diff
@@ -18,9 +18,17 @@ export default class AcceptInvitePage extends AsyncBaseContainer {
 		return await this.driver.findElement( By.css( '#email' ) ).getAttribute( 'value' );
 	}
 
-	async enterEmailAndPasswordAndSignUp( email, password ) {
+	async enterCredentialsAndSignUp( username, email, password ) {
 		await driverHelper.setWhenSettable( this.driver, By.css( '#email' ), email );
 		await driverHelper.setWhenSettable( this.driver, By.css( '#password' ), password, true );
+		// set the username field if present
+		if (
+			await driverHelper
+				.waitUntilElementLocatedAndVisible( this.driver, By.css( '#username' ), 50 )
+				.catch( () => false )
+		) {
```

Comment:
We can use a dedicated helper for that. Note that I've extended the timeout to 500ms as 50ms seem a bit low and can potentially make the test flaky.
```suggestion
		if (
			await driverHelper.isElementEventuallyLocatedAndVisible(
				this.driver,
				By.css( '#username' ),
				500
			)
		) {
```

Reply:
In this particular case, I think 50ms is enough. Because we know the page is fully rendered if the password field above was settable. I added a code comment.

Thanks a lot!

Reply:
In that case (if we don't really need to wait for it to be rendered), we should replace the `isElementEventually...` with 

```js
await driverHelper.isElementLocated( this.driver, By.css( '#username' ) );
```

Reply:
Much cleaner! Thanks ❤️